antarest/study/business/link_management.py

```diff
@@ -99,25 +99,21 @@ def update_link(self, study: RawStudy, area_from: str, area_to: str, link_update
 
         execute_or_add_commands(study, file_study, [command], self.storage_service)
 
-        updated_link = self.get_internal_link(study, link)
+        updated_link = self.get_link(study, link)
 
         return updated_link.to_dto()
 
-    def get_link_if_exists(self, file_study: FileStudy, link: LinkInternal) -> dict[str, Any]:
-        try:
-            return file_study.tree.get(["input", "links", link.area1, "properties", link.area2])
-        except KeyError:
-            raise LinkNotFound(f"The link {link.area1} -> {link.area2} is not present in the study")
-
-    def get_internal_link(self, study: RawStudy, link: LinkInternal) -> LinkInternal:
-        file_study = self.storage_service.get_storage(study).get_raw(study)
-
-        link_properties = self.get_link_if_exists(file_study, link)
-
-        link_properties.update({"area1": link.area1, "area2": link.area2})
-        updated_link = LinkInternal.model_validate(link_properties)
+    def update_all_links(
```

Comment:
I woudln't call it update_all_links as you don't necessary update all links, you can just update some of them. So update_links seems better